scripts/provision/configure_environment.sh

```diff
@@ -30,31 +30,29 @@ a2ensite magento2.conf
 sudo a2dissite 000-default
 
 # Setup PHP
-if [ ${use_php7} -eq 1 ]; then
-    sed -i "s|;include_path = \".:/usr/share/php\"|include_path = \".:/usr/share/php:${guest_magento_dir}/vendor/phpunit/phpunit\"|g" /etc/php/7.0/cli/php.ini
-    sed -i "s|display_errors = Off|display_errors = On|g" /etc/php/7.0/cli/php.ini
-    sed -i "s|display_startup_errors = Off|display_startup_errors = On|g" /etc/php/7.0/cli/php.ini
-    sed -i "s|error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT|error_reporting = E_ALL|g" /etc/php/7.0/cli/php.ini
-else
-    # Uninstall PHP 7 pre-installed in the box
-    apt-get remove -y php*
+sudo add-apt-repository ppa:ondrej/php
+sudo apt-get update
+apt-get install -y php5.6 php-xdebug php5.6-xml php5.6-mcrypt php5.6-curl php5.6-cli php5.6-mysql php5.6-gd php5.6-intl php5.6-bcmath php5.6-mbstring php5.6-soap php5.6-zip libapache2-mod-php5.6
+echo '
+xdebug.max_nesting_level=200
+xdebug.remote_enable=1
+xdebug.remote_connect_back=1' >> /etc/php/5.6/mods-available/xdebug.ini
 
-    # Install PHP 5
-    apt-get install -y php5 php5-mhash php5-mcrypt php5-curl php5-cli php5-mysql php5-gd php5-intl php5-xsl php5-xdebug curl
-    if [ ! -f /etc/php5/apache2/conf.d/20-mcrypt.ini ]; then
-        ln -s /etc/php5/mods-available/mcrypt.ini /etc/php5/apache2/conf.d/20-mcrypt.ini
-    fi
-    if [ ! -f /etc/php5/cli/conf.d/20-mcrypt.ini ]; then
-        ln -s /etc/php5/mods-available/mcrypt.ini /etc/php5/cli/conf.d/20-mcrypt.ini
-    fi
-    echo "date.timezone = America/Chicago" >> /etc/php5/cli/php.ini
+php_ini_path=/etc/php/7.0/cli/php.ini
+echo "date.timezone = America/Chicago" >> ${php_ini_path}
```

Comment:
Makes sense. Lines of code responsible for switching PHP version has been moved to "recurring"